kolibri/plugins/learn/assets/src/actions.js

```diff
@@ -1,9 +1,18 @@
-const ContentNodeResource = require('kolibri').resources.ContentNodeResource;
-const ChannelResource = require('kolibri').resources.ChannelResource;
+
+const Kolibri = require('kolibri');
+const ContentNodeResource = Kolibri.resources.ContentNodeResource;
+const ChannelResource = Kolibri.resources.ChannelResource;
+const ContentSessionLogResource = Kolibri.resources.ContentSessionLogResource;
+const ContentSummaryLogResource = Kolibri.resources.ContentSummaryLogResource;
 const constants = require('./state/constants');
 const PageNames = constants.PageNames;
 const cookiejs = require('js-cookie');
 const router = require('router');
+const intervalTimer = require('core-timer');
```

Comment:
Are you intending for this to reference an object on the core app? If so, the alias above is insufficient to do this. The core-timer will be bundled into this learn-action.js (which, in turn, is independently bundled into each renderer plugin). This means there will be at least three copies of core-timer flying around.